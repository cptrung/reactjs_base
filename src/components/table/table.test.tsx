import React from "react";
import AppTable from ".";
import { shallow } from "enzyme";
import GenerateData from "../../../stories/generateData";
import { ITableColumn } from "./type";
import { create, act } from "react-test-renderer";
import { TableCell } from "@material-ui/core";
describe("UT table component", () => {
  test("render()", () => {
    const constSchema = {
      id: "id",
      name: "name",
      address: "address",
      phone: "phone",
      birthday: "birthday",
    };
    const initialSchema = {
      [constSchema.id]: 0,
      [constSchema.name]: "",
      [constSchema.address]: "",
      [constSchema.phone]: "",
      [constSchema.birthday]: new Date(),
    };
    const generateData = new GenerateData(initialSchema);
    const data = generateData.dataEXST();
    const columns: ITableColumn[] = [
      {
        label: "id",
        key: "id",
      },
      {
        label: "name",
        key: "name",
      },
      {
        label: "address",
        key: "address",
      },
      {
        label: "phone",
        key: "phone",
      },
      {
        label: "birthday",
        key: "birthday",
      },
    ];

    const component = create(<AppTable data={data} columns={columns} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
