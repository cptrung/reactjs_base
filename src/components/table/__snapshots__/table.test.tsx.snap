// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UT table component render() 1`] = `
<div
  className="MuiPaper-root MuiTableContainer-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <table
    aria-label="simple"
    className="MuiTable-root AppTable-table-1"
  >
    <thead
      className="MuiTableHead-root"
    >
      <tr
        className="MuiTableRow-root MuiTableRow-head"
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head id AppTable-topHeaderNotGroup-6"
          scope="col"
        >
          id
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head name AppTable-topHeaderNotGroup-6"
          scope="col"
        >
          name
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head address AppTable-topHeaderNotGroup-6"
          scope="col"
        >
          address
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head phone AppTable-topHeaderNotGroup-6"
          scope="col"
        >
          phone
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head birthday AppTable-topHeaderNotGroup-6"
          scope="col"
        >
          birthday
        </th>
      </tr>
    </thead>
    <tbody
      className="MuiTableBody-root"
    >
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          88
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          4y6KhlJvbv
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          omtKe4BlkQ
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          fAChi3yhPH
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          39
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          bppyvLpjOL
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          flOLELkhfW
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          0eTm6jmPan
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          88
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          OMrBAa5Bqp
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          VZEonm078u
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          PoUCnqzrSF
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          7
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          8XLBeJT9Hh
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          iD5TxXVDek
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          jmCLs3ejC3
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          66
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          PXz3pFqEQl
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          RVqDbMYfcT
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          et7ZRFbTBv
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          59
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          fEtMPEMks6
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          QCYmvIEeac
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          hE0L2QUyjr
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          38
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          vCUYrfT19h
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          8a8MepNDMX
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          SEpP9WZe9y
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          9
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          fVuX8Km5y9
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          GZhPjSd4kZ
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          7XSFkaP4sG
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          5
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          XCeRjKaIW9
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          zRTanbCCv8
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          eUupWImY6r
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          39
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          kysxGyalWg
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          Crg136jvcD
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          B6g1O8PGw3
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          49
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          Vjtiks0Vbx
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          we7p3y4PFQ
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          UcqJy2i653
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:08.828Z
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`UT table component renderThearParent 1`] = `
<div
  className="MuiPaper-root MuiTableContainer-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <table
    aria-label="simple"
    className="MuiTable-root AppTable-table-1"
  >
    <thead
      className="MuiTableHead-root"
    >
      <tr
        className="MuiTableRow-root MuiTableRow-head"
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head id AppTable-topHeaderNotGroup-6"
          scope="col"
        >
          id
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head name AppTable-topHeaderNotGroup-6"
          scope="col"
        >
          name
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head address AppTable-topHeaderNotGroup-6"
          scope="col"
        >
          address
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head phone AppTable-topHeaderNotGroup-6"
          scope="col"
        >
          phone
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head birthday AppTable-topHeaderNotGroup-6"
          scope="col"
        >
          birthday
        </th>
      </tr>
    </thead>
    <tbody
      className="MuiTableBody-root"
    >
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          22
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          7VR59yZdMi
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          cq6Ckhv96P
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          CBFCT8WaUk
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          64
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          PDGHYbp2RT
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          NpDdY4Ew6o
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          8NQQokbzTK
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          63
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          VwcMDq33kG
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          uMQlaDVdyU
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          jtSPbHBSWl
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          11
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          rCQL6EewpK
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          vV7z0CeOHp
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          BN4mDYzXjf
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          36
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          MtI5MKK6Wc
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          CVzUP08vYt
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          dzXh8yS6O4
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          Fj352Di6R7
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          4EbI9LWiK6
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          yGO59kTYht
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          16
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          zA69BBjqf9
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          hWI1GSqvUj
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          Q2r0FTHEY6
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          59
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          SQliXx52fP
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          QBzLlzFtiE
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          tTuWfj37pE
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          73
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          EzbIMqG5pm
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          rj62spA1GE
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          rexO6d8cUu
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          10
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          C61xCNqoLD
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          5WCjxpfoot
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          kx6igHAiVW
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        onClick={[Function]}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body id"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          65
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body name"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          byYLGmfYdI
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body address"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          qfPkj5DRdM
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body phone"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          rSI7XoTqpA
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body birthday"
          colSpan={1}
          onClick={[Function]}
          rowSpan={1}
          style={Object {}}
        >
          2020-03-23T10:13:00.588Z
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
